.travis.yml

```diff
@@ -4,6 +4,8 @@ script:
   - cargo build --verbose
   - cargo test --verbose
   - cargo doc
+  - cd regex_macros
+  - cargo test --verbose
```

Comment:
Ah and with benchmarks to run we may want to consider running `cargo bench` here as well (just to make sure things compile)